### PR TITLE
Updates needed to make create_release_archives.sh work from a mac

### PR DESCRIPTION
**Pull request by ldemailly**
With brew install gnu-tar and wget:

TAR=gtar CP=gcp sh -x ./release/create_release_archives.sh
Works

### CODE REVIEW

**Comment by ldemailly** [comment]
@rshriram mac (bsd) `tar` and `cp` do not understand  `--owner` and `--parent` respectively which are GNU extensions

File: release/create_release_archives.sh [hunk 1]
```diff
@@ -34,6 +34,16 @@ COMMON_FILES_DIR="${TMP_DIR}/istio/istio-${ISTIO_VERSION}"
...
 mkdir -p "${COMMON_FILES_DIR}/bin" "${ARCHIVES_DIR}"
 
+# On mac, brew install gnu-tar gnu-cp
+# and set CP=gcp TAR=gtar
+
+if [[ -z "${CP}" ]] ; then
+  CP=cp
+fi
+if [[ -z "${TAR}" ]] ; then
+  TAR=gtar
```
Comment (sebastienvas) [inline release/create_release_archives.sh]: Did you mean to use tar here ?

Comment (sebastienvas) [inline release/create_release_archives.sh]: Otherwise that would break linux users.

Comment (ldemailly) [inline release/create_release_archives.sh]: you are right ! problem with only testing on my box :-) (though I think it's frequent to also have gtar on linux)

**Pull request merged**